packages/salesforcedx-vscode-apex/.vscodeignore

```diff
@@ -20,10 +20,13 @@ node_modules/@salesforce/salesforcedx-utils-vscode/webpack.config.js
 node_modules/@salesforce/salesforcedx-utils-vscode/out/test/**
 
 # salesforcedx-sobjects-faux-generator files to ignore
+node_modules/@salesforce/salesforcedx-sobjects-faux-generator/node_modules/**
+!node_modules/@salesforce/salesforcedx-sobjects-faux-generator/node_modules/vscode-nls/**/*
```

Comment:
When bundling the apex vsix file, we ignore everything under `salesforcedx-sobjects-faux-generator/node_modules` except for the `vscode-nls` library since we can't process it using webpack.